tools/run_tests/run_interop_tests.py

```diff
@@ -763,15 +766,23 @@ def cloud_to_prod_jobspec(language,
                           docker_image=None,
                           auth=False,
                           manual_cmd_log=None,
-                          service_account_key_file=None):
+                          service_account_key_file=None, 
+                          transport_security='tls'):
     """Creates jobspec for cloud-to-prod interop test"""
     container_name = None
     cmdargs = [
         '--server_host=%s' % server_host,
         '--server_host_override=%s' % server_host, '--server_port=443',
-        '--use_tls=true',
         '--test_case=%s' % test_case
     ]
+    if transport_security == 'tls':
+        transport_security_options += ['--use_tls=true']
+    elif transport_security == 'google_default'
+        transport_security_options += ['--use_google_default=true']
```

Comment:
any reason to not just pass `--custom_credential_type=google_default_credentials`  here? I think that would allow removal of the `--use_google_default` logic within the interop client.